File: 351004/Hasiuk/Labs/src/main/java/com/example/Labs/dto/response/StoryResponseTo.java
package com.example.Labs.dto.response;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class StoryResponseTo {
    private Long id;
    private Long editorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}

File: 351002/vorobei/Publisher/BusinessLogic/DTO/Response/StoryResponseTo.cs
using System.Text.Json.Serialization;
using DataAccess.Models;

namespace BusinessLogic.DTO.Response
{
    public class StoryResponseTo : BaseEntity
    {
        [JsonPropertyName("creatorId")]
        public int CreatorId { get; set; }

        [JsonPropertyName("title")]
        public string Title { get; set; } = string.Empty;

        [JsonPropertyName("content")]
        public string Content { get; set; } = string.Empty;

        [JsonPropertyName("created")]
        public DateTime Created { get; set; }

        [JsonPropertyName("modified")]
        public DateTime Modified { get; set; }
    }
}
